lib/migration.js

```diff
@@ -186,7 +186,7 @@ function mixinMigration(MySQL, mysql) {
     function actualize(propName, oldSettings) {
       var newSettings = m.properties[propName];
       if (newSettings && changed(newSettings, oldSettings)) {
-        var pName = self.client.escapeId(propName);
+        var pName = self.columnEscaped(model, propName);
```

Comment:
change this to `var pName = (newSettings.mysql && newSettings.mysql.columnName) || self.client.escapeId(propName);`